lib/tool_belt/pulp_repository_updater.rb

```diff
@@ -34,21 +35,21 @@ def compare_repos(os_version, client = false)
       katello_repo = client ? katello_client_repo(os_version) : katello_pulp_repo(os_version)
       setup_pulp_repo(os_version)
       command = "repodiff --simple --old=#{katello_repo} --new=file://#{Dir.pwd}/tmp/"
-      @systools.execute(command)[0]
+      @systools.syscall(command)[0]
     end
 
     def setup_pulp_repo(os_version)
-      @systools.execute('rm -rf tmp/')
+      @systools.syscall('rm -rf tmp/')
       Dir.mkdir('tmp')
       Dir.chdir('tmp') do
-        syscall("wget #{pulp_repo(os_version)}")
+        @systools.syscall("wget #{pulp_repo(os_version)}")
         index = File.read('index.html')
         rpms = index.scan(/href=".*.rpm"/).collect { |link| link.split('href=')[1].gsub('"', '') }
         rpms.each do |rpm|
-          syscall("wget #{pulp_repo(os_version)}#{rpm}")
+          @systools.syscall("wget #{pulp_repo(os_version)}#{rpm}")
         end
-        syscall("sudo yum -y install createrepo") unless syscall('rpm -q createrepo')[1]
-        syscall("createrepo .")
+        @systools.syscall("sudo yum -y install createrepo") unless @systools.syscall('rpm -q createrepo')[1]
+        @systools.syscall("createrepo .")
```

Comment:
Actually it should be used here, as you need to create repo in order to compare the repos